app/views/lists/_index.html.erb

```diff
@@ -1,25 +1,35 @@
-<% lists.each do |list| %>
-  <div class="card">
-    <% if user_signed_in? %>
-      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
-    <% end %>
-    <span class="info">
-      <h2><%= list.name %></h2>
-      <p><%= list.description %></p>
-      <p>Tags:<i><%= list.tag_list %></i></p>
-    </span>
-    <span class="content">
-      <%= render partial: 'lists/references', locals: { list: list } %>
-    </span>
-    <span class="options">
-      <ul>
-        <li><%= link_to 'Show', list %></li>
+<% lists.each_slice(3) do |three_lists| %>
```

Comment:
ok ill give it a shot.

On Fri, Aug 12, 2016 at 1:59 PM, Brian Kung <notifications@github.com>
wrote:

> In app/views/lists/_index.html.erb
> <https://github.com/briankung/projectcredo/pull/52#discussion_r74643513>:
>
> > -  <div class="card">
> > -    <% if user_signed_in? %>
> > -      <%= render partial: 'votes/toggle_like', locals: { votable: list } %>
> > -    <% end %>
> > -    <span class="info">
> > -      <h2><%= list.name %></h2>
> > -      <p><%= list.description %></p>
> > -      <p>Tags:<i><%= list.tag_list %></i></p>
> > -    </span>
> > -    <span class="content">
> > -      <%= render partial: 'lists/references', locals: { list: list } %>
> > -    </span>
> > -    <span class="options">
> > -      <ul>
> > -        <li><%= link_to 'Show', list %></li>
> > +<% lists.each_slice(3) do |three_lists| %>
>
> If you set the width to 4 columns or whatever (out of 12)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/52/files/b653b00e5e4f23efb6cb495404de572981f0e34c#r74643513>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZKbEgzrSAIMg1wslob-dimdLdPfQks5qfMJ3gaJpZM4JjW-e>
> .
>